x-pack/plugin/logsdb/src/javaRestTest/java/org/elasticsearch/xpack/logsdb/LogsIndexModeEnabledRestTestIT.java

```diff
@@ -258,8 +258,8 @@ public void testLogsAtSettingWithTimeSeriesOverride() throws IOException {
                   "index": {
                     "routing_path": [ "hostname" ],
                     "mode": "time_series",
-                    "sort.field": [],
-                    "sort.order": []
+                    "sort.field": ["_tsid", "@timestamp"],
```

Comment:
Shouldn't the sort.field / sort.order index settings be derived from the defaults? Or maybe the settings shouldn't be defined in this test?